src/WingetCreateCLI/Commands/UpdateCommand.cs

```diff
@@ -138,6 +138,22 @@ public override async Task<bool> Execute()
                     return false;
                 }
 
+                var argumentsRequiringSubmit = new List<string> { nameof(this.PRTitle), nameof(this.Replace) };
+                bool submissionArgumentsUsed = !string.IsNullOrEmpty(this.PRTitle) || this.Replace;
+
+                if (submissionArgumentsUsed && !this.SubmitToGitHub)
+                {
+                    Logger.ErrorLocalized(nameof(Resources.SubmitFlagRequired_ErrorMessage));
+                    Console.WriteLine();
+
+                    foreach (string argument in argumentsRequiringSubmit)
+                    {
+                        Logger.Error($"--{argument.ToLower()}");
+                    }
+
+                    return false;
+                }

```

Comment:
I think this would be better to simply check the condition:
`(!this.SubmitToGitHub && (!string.IsNullOrEmpty(this.PRTitle) || this.Replace))`

Then only warn the user and make it not blocking like so:

`Submit arguments were provided. Did you forget to include the --submit, -s flag?`

Our current behavior is to not make it blocking if the -s flag is not provided and continue outputting an updated manifest so we should adhere to that. Your check and warning would be an improvement though so that we can notify the user if they forget an arg. 

Be sure to also add a unit test to verify that this warning message is outputted. You can look at the existing examples in WingetCreateTests/UnitTests/UpdateCommandTests.cs to see how we check if a string output is correctly shown.

Reply:
Made the behavior non-blocking and added a unit test for checking the output